bundles/org.openhab.ui/web/src/pages/settings/rules/rule-edit.vue

```diff
@@ -1,11 +1,14 @@
 <template>
   <f7-page @page:afterin="onPageAfterIn" @page:afterout="onPageAfterOut">
     <f7-navbar :title="isNewRule ? 'Create rule' : rule.name" back-link="Back" no-hairline>
-      <f7-nav-right v-if="isEditable">
-        <f7-link @click="save()" v-if="$theme.md" icon-md="material:save" icon-only />
-        <f7-link @click="save()" v-if="!$theme.md">
-          Save<span v-if="$device.desktop">&nbsp;(Ctrl-S)</span>
-        </f7-link>
+      <f7-nav-right>
+        <developer-dock-icon />
```

Comment:
As long as we don't have extra content for the rule edit page to show in the sidebar, I would not add the help icon there.

Reply:
@jimtng WDYT?

Reply:
Perhaps we should add an extra (help) content?

It's still handy to a) lookup items, b) create a new item using the dev sidebar

Reply:
> Perhaps we should add an extra (help) content?

In case you have an idea what content to show there, it would be great if you could add that content.
I can provide help regarding the conventions required by the UI to properly render that Markdown file.

> It's still handy to a) lookup items, b) create a new item using the dev sidebar

Agreed, so I will add it back to rule-edit and merge.